### PR TITLE
Adding @jack-berg and @trask as codeowners

### PR DESCRIPTION
**Pull request by breedx-splk**
We talked about this in the last SIG and @jack-berg and @trask agreed to be `CODEOWNERS` for a period to help get the project going and reduce time-to-merge across timezones.

### CODE REVIEW

**Comment by jack-berg** [comment]
I've went ahead and created @open-telemetry/android-approvers 

Skipped making it when I initialized the repo because we didn't have any approvers.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -3,3 +3,5 @@
 
  # For anything not explicitly taken by someone else:
 * @open-telemetry/android-maintainers
+* @trask
+* @jack-berg
```
Comment (jack-berg) [inline .github/CODEOWNERS:7]:
```suggestion
* @open-telemetry/android-approvers
```

**Comment by trask** [comment]
@breedx-splk probably worth adding something like https://github.com/open-telemetry/opentelemetry-java-instrumentation#contributing, which is what we PR against when adding/removing approvers/etc

**Comment by breedx-splk** [comment]
> @breedx-splk probably worth adding something like https://github.com/open-telemetry/opentelemetry-java-instrumentation#contributing, which is what we PR against when adding/removing approvers/etc

Yeah agreed. I'll add that shortly! Thanks!

**Review by trask — APPROVED** [review]

**Review by jack-berg — APPROVED** [review]

**Pull request merged**